Discord.Net.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
-VisualStudioVersion = 15.0.26228.4
+VisualStudioVersion = 15.0.26730.12
```

Comment:
I don't know if changes to .sln file will cause merge conflicts or not. Might be easier to exclude this file?

Reply:
The person who merge the PR will just have to ignore this during merge